src/cli.ts

```
@@ -0,0 +1,156 @@
+        } else {
+          console.error(`An unknown error occurred during analysis: ${String(error)}`);
+        }
+        process.exit(1);
+      }
+    });
+
+  // Use parseAsync to correctly handle asynchronous command actions
+  await program.parseAsync(process.argv);
+
+  // If no commands or arguments are provided, show help
+  if (!process.argv.slice(2).length) {
```

ESLint

[error] Unexpected use of the global variable 'process'. Use 'require("process")' instead.

(node/prefer-global/process)